doc/user/content/installation/install-on-aws/_index.md

```diff
@@ -117,20 +117,46 @@ for evaluation purposes only. The modules deploy a sample infrastructure on AWS
 1. Create a `terraform.tfvars` file (you can copy from the
    `terraform.tfvars.example` file) and specify the following variables:
 
+   {{< tabs >}}
+   {{< tab "Without cert-manager" >}}
+
```

Comment:
https://preview.materialize.com/materialize/32065/self-managed/v25.1/installation/install-on-aws/#set-up-aws-kubernetes-environment-and-install-materialize

Reply:
Oh, just saw the slack message about enabling TLS by default. Will update a patch.